v3/util/oid.go

```diff
@@ -53,6 +53,7 @@ var (
 	SubjectInfoAccessOID         = asn1.ObjectIdentifier{1, 3, 6, 1, 5, 5, 7, 1, 11}       // Subject Info Access Syntax
 	SubjectKeyIdentityOID        = asn1.ObjectIdentifier{2, 5, 29, 14}                     // Subject Key Identifier
 	ReasonCodeOID                = asn1.ObjectIdentifier{2, 5, 29, 21}                     // CRL Reason Code
+  CRLNumberOID                 = asn1.ObjectIdentifier{2, 5, 29, 20}                     // CRL Number
```

Comment:
Sorry, I fixed a merge conflict via the GitHub UI, but it decided to use spaces instead of tabs (despite my inputs) which broke the linter.

```suggestion
	CRLNumberOID                 = asn1.ObjectIdentifier{2, 5, 29, 20}                     // CRL Number
```